<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bgwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1190</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgwidget{background-color: rgb(189, 236, 182);}</string>
    </property>
    <widget class="QLineEdit" name="ejey_lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>550</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ejex_lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>500</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="moduloButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>450</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255);
border-style: outset;
border-width: 2px;
border-color: black;
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>Módulo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="faseButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>450</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255);
border-style: outset;
border-width: 2px;
border-color: black;
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>Fase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ocultar_medicion_pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>390</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(224, 177, 65);
border-radius:15px;

border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>Ocultar</string>
     </property>
    </widget>
    <widget class="QLabel" name="ejeylabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>550</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Eje Y:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Curva 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="medicionButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(224, 177, 65);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>MEDICIÓN</string>
     </property>
    </widget>
    <widget class="QLabel" name="ejexlabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>500</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Eje X:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ocultar_teorico_Button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(88, 224, 173);
border-radius:15px;

border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>Ocultar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ocultar_simulacion_pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(205, 139, 224);

border-radius:15px;

border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>Ocultar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="simulacionButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(205, 139, 224);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>SIMULACIÓN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="borrarpushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>610</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(148, 119, 50);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>BORRAR
GRÁFICOS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Curva 3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrar_medicion_pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(224, 177, 65);
border-radius:15px;
border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>Mostrar</string>
     </property>
    </widget>
    <widget class="MplWidget" name="MplWidget2" native="true">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>440</y>
       <width>680</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="teoricoButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(88, 224, 173);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>TEORICO</string>
     </property>
    </widget>
    <widget class="MplWidget" name="MplWidget" native="true">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>120</y>
       <width>680</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrar_simulacion_pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(205, 139, 224);

border-radius:15px;
border-width: 2px;
border-color: black;</string>
     </property>
     <property name="text">
      <string>Mostrar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Curva 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrar_teorico_Button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(88, 224, 173);
border-width: 2px;
border-color: black;
border-radius:50px;</string>
     </property>
     <property name="text">
      <string>Mostrar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
